icu4c/packaging/distrelease.ps1

```diff
@@ -13,7 +13,7 @@
 #  (bring up Powershell ISE)
 #    cd C:\icu\icu4c\
 #    Set-ExecutionPolicy -Scope Process Unrestricted
-#    .\packaging\distrelease.ps1
+#    .\packaging\distrelease.ps1 "x64 or x86"
```

Comment:
What about:
`.\packaging\distrelease.ps1 -arch x64`
or
`.\packaging\distrelease.ps1 -arch x86`
?

TBH, not sure, but it seems that using named parameters is more "PowerShelly".

Reply:
I'll update it since it's clearer as well.